apps/api/src/common/constants/database.ts

```diff
@@ -2,3 +2,8 @@ export const Status = {
   INACTIVE: 0,
   ACTIVE: 1,
 };
+
+export const Roles = {
```

Comment:
Is this needed as part of this PR? If not, we can remove it for now and add it later on when we need it.

Reply:
Will remove this as not needed currently